### PR TITLE
[bitnami/rabbitmq] Allow for templating in extraSecrets

### PR DESCRIPTION
**Pull request by while-loop**
**Description of the change**

Allow for the `extraSecrets` value to be templated.

**Benefits**

Allows for dynamic values when supplying a load-definition:

Input: 
```yaml
extraSecrets:
  load-definition:
    load_definition.json: |
        {
          "vhosts": [
            {
              "name": "/"
            }
          ],
          "users": [
            {
              "name": "{{ .Values.auth.username}}",
              "password": "{{ .Values.auth.password}}",
              "tags": "administrator"
            }
          ],
          "permissions": [
            {
              "user": "{{ .Values.auth.username}}",
              "vhost": "/",
              "configure": ".*",
              "write": ".*",
              "read": ".*"
            }
          ],
          "policies": [
            {
              "name": "ha-all",
              "apply-to": "all",
              "pattern": ".*",
              "vhost": "/",
              "definition": {
                "ha-mode": "all",
                "ha-sync-mode": "automatic"
              }
            }
          ]
        }
```

Output:
```json
{
  "vhosts": [
    {
      "name": "/"
    }
  ],
  "users": [
    {
      "name": "something",
      "password": "something",
      "tags": "administrator"
    }
  ],
  "permissions": [
    {
      "user": "something",
      "vhost": "/",
      "configure": ".*",
      "write": ".*",
      "read": ".*"
    }
  ],
  "policies": [
    {
      "name": "ha-all",
      "apply-to": "all",
      "pattern": ".*",
      "vhost": "/",
      "definition": {
        "ha-mode": "all",
        "ha-sync-mode": "automatic"
      }
    }
  ]
}
```

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [x] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/rabbitmq/templates/secrets.yaml [hunk 1]
```diff
@@ -31,5 +31,5 @@ metadata:
   namespace: {{ $.Release.Namespace }}
   labels: {{- include "common.labels.standard" $ | nindent 4 }}
 type: Opaque
-stringData: {{- $value | toYaml | nindent 2 }}
+stringData: {{- tpl ($value | toYaml | nindent 2) $ }}
```
Comment (FraPazGal) [inline bitnami/rabbitmq/templates/secrets.yaml:34]:
```suggestion
stringData: {{- include "common.tplvalues.render" (dict "value" $value "context" $) | nindent 2 }}
```

**Review by FraPazGal — CHANGES_REQUESTED** [review]
Hi @while-loop,

Thanks for the PR!! Just a minor change to use our `common` helpers library. Also, please include `(evaluated as a template)` for the `extraSecrets` parameter's value in the README.

**Comment by while-loop** [comment]
Thanks for the requested changes @FraPazGal!

**Review by FraPazGal — APPROVED** [review]
LGTM! Thanks again!

**Pull request merged**